package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "mv_weatherapp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
You can also delete the test script here since it's not being used. 
Its good practice to add a `dev` and a `build` script. So in each one, you can specify in which mode you want to Webpack run. Something like:
`"dev": "webpack --mode development"`
`"build": "webpack --mode production"`
The README should be updated accordingly with instructions about each script.
![Screenshot from 2020-04-03 08-46-59](https://user-images.githubusercontent.com/30977845/78357352-b40ce480-7587-11ea-9bf0-9a19caeef588.png)